ui/app/models/auth-config/ldap.js

```diff
@@ -9,7 +9,7 @@ export default AuthConfig.extend({
   useOpenAPI: true,
   certificate: attr({
     label: 'Certificate',
-    editType: 'textarea',
+    editType: 'file',
```

Comment:
This is a certificate and needed to be a file upload with option to add as text with masked input.